examples/.experimental/next-app-dir/src/trpc/server-invoker.ts

```diff
@@ -1,35 +1,31 @@
 import { loggerLink } from '@trpc/client';
 import { experimental_nextCacheLink } from '@trpc/next/app-dir/links/nextCache';
 import { experimental_createTRPCNextAppDirServer } from '@trpc/next/app-dir/server';
-import { auth } from '~/auth';
 import { appRouter } from '~/server/routers/_app';
-import { cookies } from 'next/headers';
 import superjson from 'superjson';
+import { createContext } from './shared-server';
 
 /**
  * This client invokes procedures directly on the server without fetching over HTTP.
  */
 export const api = experimental_createTRPCNextAppDirServer<typeof appRouter>({
   config() {
     return {
+      createContext: () => createContext('invoke'),
+      contextSelector: (ctx, callOpts) => {
```

Comment:
I think this example should show a "private by default" and opt-in to sharing cache for specific procedures thst doesnt have sensitive data

Reply:
Good thoughts, will implement